src/main/java/me/clip/placeholderapi/commands/impl/cloud/CommandECloudDownload.java

```diff
@@ -55,6 +55,12 @@ public void evaluate(@NotNull final PlaceholderAPIPlugin plugin,
       return;
     }
 
+    if (!expansion.isVerified() && !plugin.getPlaceholderAPIConfig().cloudAllowUnverifiedExpansions()) {
+      Msg.msg(sender, "&cThe expansion '&f" + params.get(0) + "&c' is not verified and can only be downloaded manually from &fhttps://api.extendedclip.com");
+      Msg.msg(sender, "&aTo allow the download of unverified expansions set &fcloud_allow_unverified_expansions: true &ain config.yml");
```

Comment:
Given that this only happens with expansions that aren't verified, I doube the problem is that big here.

After all will people usually stick to the ol' reliable ones like Player or Server expansion...
But given the current situation regarding verification of expansions (That every new release results in them being auto-unverified), is this imo a change needed as otherwise people will yell about bugs to expansions devs that have been fixed already. I speak from experience.

Reply:
The change on the ecloud wasn't made just to be easily nullified tho.

Reply:
Thinking about it, you might be right, Blitz, the first message is foolproof enough. Plus, you don't download expansions every day. A simple message telling you that the expansion exists, but it is not verified due to various reasons, but you can still download it manually gives enough info.

And @Andre_601 the expansions are usually re-verified after a short period of time.

Reply:
Yeah I think it would be smart to not advertise the config option - expansions are unverified for a reason, namely safety, and disabling this by wanton server owners blindly is just going to be more cause for problems.